### PR TITLE
[Clean Up] : unexpected end of file

### PR DESCRIPTION
**Pull request by ro4i7**
## Purpose / Description
fix warning of property naming conversion

## Fixes
* Changes towards #13282 

## Approach
emulator

## How Has This Been Tested?
emulator

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison — CHANGES_REQUESTED** [review]
The commit does not match the commit message

**Review by david-allison — APPROVED** [review]
Looks good, thanks!

File: annotations/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Annotations should now appear
 ## Modificiation Instructions
```
Comment (lukstbit) [inline annotations/README.md:25]: Could you please also fix the typo for this word?

File: annotations/README.md [hunk 2]
```diff
@@ -46,4 +46,4 @@ Sample:
 
 It would be ideal if these could be set on a per-project basis. I haven't had the time to determine whether this is possible.
 
-These annotations are not yet supported by our automated tooling.
\ No newline at end of file
+These annotations are not yet supported by our automated tooling.
```
Comment (lukstbit) [inline annotations/README.md:49]: Why was this line changed?

**Review by lukstbit — CHANGES_REQUESTED** [review]
Thank you for contributing, I've left two requests and then we could merge this.

File: annotations/README.md (hunk 2 above)
Comment (ro4i7) [reply annotations/README.md:49]: No, it's same as original one.

Comment (lukstbit) [reply annotations/README.md:49]: Yes, you added a new line at the end and Github showed it as changed.

**Review by lukstbit — APPROVED** [review]
All good, thanks for making the change!

**Pull request merged**